e2e_tests/tests/per_provider/normal_tests/auth.rs

```diff
@@ -12,10 +12,16 @@ fn two_auths_same_key_name() -> Result<()> {
     let auth2 = String::from("second_client");
 
     client.set_default_auth(Some(auth1));
+    #[cfg(not(feature = "cryptoauthlib-provider"))]
     client.generate_rsa_sign_key(key_name.clone())?;
 
     client.set_default_auth(Some(auth2));
-    client.generate_rsa_sign_key(key_name)
+    #[cfg(not(feature = "cryptoauthlib-provider"))]
+    let result = client.generate_rsa_sign_key(key_name.clone());
+    #[cfg(feature = "cryptoauthlib-provider")]
+    let result = client.generate_ecc_key_pair_secpr1_ecdsa_sha256(key_name.clone());
```

Comment:
To make the test useful, can you also add those two lines after line 16? Basically here we are creating two keys with the same names but different app names to check that it works correctly.

Reply:
Accepted.